### PR TITLE
Feat : Added helper to generate suggestions

### PR DESCRIPTION
**Pull request by ardourApeX**
### Description:
1. Added helper to generate sentence based on provided parameters

### Anything you would like to inform the reviewer about:
NA

### Dev Tested:
- [x] Yes

### Under Feature Flag:
- [x] Yes

### Frontend Changes:
- [x] Yes

### Images/video of the change:

NA

### Test Suits
image
image
image

### Test Coverage
image
image
image

### CODE REVIEW

**Comment by skv93-coder** [comment]
LGTM

File: src/interfaces/searchOptions.type.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export interface IOption {
```
Comment (ajoykumardas12) [inline src/interfaces/searchOptions.type.ts:1]: The name can be more descriptive I think. The name suggests I as abbreviation but not clear what.

File: src/utils/generateSuggestions.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            (field !== 'assignee' || matchedOptions['assignee'] === userInput)
+        ) {
+            continue;
+        } else {
+            if (
+                additionalCheck &&
+                (field === 'title' || field === 'assignee')
+            ) {
+                const value = {
+                    [field]: typedValue.trim().toLowerCase(),
```
Comment (ajoykumardas12) [inline src/utils/generateSuggestions.ts:46]: Searching by Title is case-sensitive.

File: src/interfaces/searchOptions.type.ts (hunk 1 above)
Comment (ardourApeX) [reply src/interfaces/searchOptions.type.ts]: I changed it to `TaskSearchOption`. I couldn't come up with a better name than this.

File: src/utils/generateSuggestions.ts (hunk 2 above)
Comment (ardourApeX) [reply src/utils/generateSuggestions.ts]: Done

**Review by iamitprakash — APPROVED** [review]

**Pull request merged**